pkg/api/controller.go

```diff
@@ -4830,3 +4836,14 @@ func encodeGCUncommittedMark(mark *catalog.GCUncommittedMark) (*string, error) {
 	token := base64.StdEncoding.EncodeToString(raw)
 	return &token, nil
 }
+
+func extractLakeFSMetadata(header http.Header) map[string]string {
```

Comment:
1. We can pass the prefix and this function will be more generic (extractMetadataByHeaderPrefix)
2. The metadata key should be without the prefix to make this one general solution for all the entry's metadata.

Reply:
Please look again - I've made some adjustments